src/fastfield/bytes/writer.rs

```diff
@@ -129,6 +129,7 @@ impl BytesFastFieldWriter {
         } else {
             value_serializer.write_all(&self.vals)?;
         }
+        value_serializer.flush()?;
```

Comment:
I have no clue what this is about. Can you open a separate PR?

Reply:
I forgot to remove that, I was investigating an issue which was rooted in composite_file.rs. I'll add a longer explanation to the PR commit message about the issue.